lib/child-writer/index.js

```diff
@@ -127,6 +127,7 @@ exports.write = (image, drive, options) => {
    */
   const bridgeRobotMessage = (message) => {
     const ROBOT_COMMAND_LOG = 'log';
+    const ROBOT_COMMAND_ERROR = 'error';
```

Comment:
Agree; `robot.COMMAND_LOG` would follow the usual pattern of how constants come from places